launcher/src/functionalTest/kotlin/com/cognifide/gradle/aem/launcher/LauncherTest.kt

```diff
@@ -1,25 +1,35 @@
 package com.cognifide.gradle.aem.launcher
 
 import org.buildobjects.process.ProcBuilder
-import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 import java.io.File
+import java.util.*
 import java.util.concurrent.TimeUnit
 
 class LauncherTest {
 
     @Test
-    fun shouldRunProperly() = test("tasks") { launch("tasks") }
+    fun shouldRunProperly() = makeTestProject("tasks", null) { launch("tasks") }
 
     @Test
-    fun shouldDisplayInstanceStatus() = test("instance-status") {
+    fun shouldDisplayInstanceStatus() = makeTestProject("instance-status", "6.5.10") {
```

Comment:
I was asking you about introducing another test case for "cloud" aemVersion and making assertions / running some task like `environmentResolve` to prove that scaffolder generated what was needed in particular case